source/deploying-with-kubernetes/kubernetes-clean-up.rst

```diff
@@ -0,0 +1,261 @@
+.. Copyright (C) 2018 Wazuh, Inc.
+
+.. _kubernetes_clean_up:
+
+
+Clean Up
+========
+
+Steps to perform a clean up of our deployments, services and volumes used in our environment.
+
+Wazuh managers
+--------------
+
+The deployment of the Wazuh cluster of managers involves the use of various statefulSet elements as well as configuration maps and services.
+
+1. The first step is to remove the pods corresponding to the managers.
+
```

Comment:
It is not necessary to delete the pods, when the sts are deleted they will be deleted. I miss to put it in the previous revision.